cql3/update_parameters.hh

```diff
@@ -162,7 +154,7 @@ public:
     }
 
     gc_clock::duration ttl() const {
-        return _ttl.count() > 0 ? _ttl : _schema->default_time_to_live();
+        return _ttl.has_value() ? _ttl.value() : _schema->default_time_to_live();
```

Comment:
nit: ._ttl.value_or(_schema->default_time_to_live()) should be more efficient I think.

Reply:
Why would it be more efficient? It should do exactly the same thing. But I agree it's shorter so I'll change it.

Reply:
Getting `_ttl.value()` checks again if it's engaged and throws otherwise (presuming no inlining optimizations), for `value_or`, it's sufficient to check once.

Reply:
Wow, you're right - I was always under the impression that value() was just another name for operator*, I didn't realize it did extra checking - which is almost always not needed :-(